### ISSUE TITLE
Publish VCF files

### ISSUE THREAD

**Issue by hoelzer** [replikation/poreCov#266]
For many downstream analysis it would be nice to also publish the VCF files with the ref/alternate allele and the frequencies.

**Comment by hoelzer** [comment]
Ah wait, it seems we publish them to `3.Lineages_Clades_Mutations` folder

E.g.

* BM_IMSSC2-80-2024-00028.fail.vcf
* SNP_BM_IMSSC2-80-2024-00028.pass.vcf

are the final variant calls splitted into pass and fail? (what are the default filter values then?)

**Comment by MarieLataretu** [comment]
Yes, the called variants get split based on `min_depth` (DP) and `min_depth` (QUAL and GQ):
https://github.com/artic-network/fieldbioinformatics/blob/1.3.0-dev/artic/vcf_filter.py#L54-L79

The default values in poreCov are 20 for `min_depth` and `${params.min_depth} = 20` for `min_depth` - the same as in ARTIC (https://github.com/artic-network/fieldbioinformatics/blob/1.3.0-dev/artic/pipeline.py#L115-L118).

Note, that `min_depth` and `min_depth` are not available in ARTIC's master branch, only in 1.3.0-dev.

@jonas-fuchs pointed out that VCFs from ARTIC in Galaxy do have allele frequencies. I didn't have time to test it, but it'd be a `medaka tools annotate` with additional `--dpsp` and [this script](https://github.com/galaxyproject/tools-iuc/blob/main/tools/medaka/convert_VCF_info_fields.py).
